src/components/Sidebar/components/Backdrop/Backdrop.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
```

Comment:
I just had a general thought about this component. We need a `Backdrop` or `Overlay` in multiple contexts. For example, our modals also have an overlay. Now, the modals use `react-modal` and the styles for the overlay there are passed differently. But maybe we can extract those styles and share them between this component here and the modals? Just an idea to get consistency.

Reply:
Great idea! I can work on it in a future PR, what do you think? 😃